### PR TITLE
04 html tables

### PR DESCRIPTION
**Pull request by guillaume-boutin**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,11 +37,29 @@
...
             <h2>Schedule</h2>
+            <table>
+              <tr>
+                <th>Morning</th>
+                <th>Noon</th>
+                <th>Afternoon</th>
+                <th>Afternoon break</th>
+                <th>Late afternoon</th>
+                <th>Evening</th>
+              </tr>
```
Comment (nyanofthemoon) [inline index.html]: @guillaume-boutin Please review documentation for `th` versus `td` relating to table headers.

File: index.html [hunk 2]
```diff
@@ -72,11 +37,29 @@
...
         <section class="schedule">
             <h2>Schedule</h2>
+            <table>
+              <tr>
+                <th>Morning</th>
+                <th>Noon</th>
+                <th>Afternoon</th>
+                <th>Afternoon break</th>
+                <th>Late afternoon</th>
+                <th>Evening</th>
```
Comment (nyanofthemoon) [inline index.html]: @guillaume-boutin We don't want to ruin your creativity, but once the requirements for a project are set, it is important to respect them. Please remember that employers are going to look at your Github.

**Comment by guillaume-boutin** [comment]
I'm not sure what else there still is to fix. A hint?

**Comment by nyanofthemoon** [comment]
@guillaume-boutin Please refer to https://github.com/DecodeMTL/html-css-intro under Exercise #4 under "Complete the "Schedule" section of the page so that it looks like this:".

**Comment by nyanofthemoon** [comment]
@guillaume-boutin Close enough- thank you! :+1:

**Pull request closed without merging**